docs/docs/target/kubernetes.md

```diff
@@ -28,9 +28,50 @@ Kubernetes resource definition is scanned for:
 
 
 !!! warning "Important Notice"
-     To successfully scan a Kubernetes cluster, `trivy kubernetes` subcommand must be executed under a role that has read permissions at the cluster scope.
-
-     Trivy must be able to access information about all cluster resources, including pods, deployments etc.
+     To successfully scan a Kubernetes cluster, `trivy kubernetes` subcommand must be executed under a role that has some specific permissions.
+
+     Without `node collecor` (with flag `--disable-node-collector`) the role must have `list` verb for all resources ("*") inside next API groups: core (""), "apps", "batch","networking.k8s.io", "rbac.authorization.k8s.io":
```

Comment:
The call-outs box is a bit long. I think we can add a new section for required roles.

![CleanShot 2024-09-04 at 09 56 47](https://github.com/user-attachments/assets/946d406b-5eac-4f98-b658-76316a8a4e72)

Reply:
done